### DISCUSSION TITLE
Load model from local path

### DISCUSSION THREAD

**Discussion by lanesket** [mindee/doctr#1215] [Q&A] [answered]
Is there any way to download the model (`db_resnet50`) from the directory (without Hub) if I already have the weights downloaded, this is needed to use the model on a computer without internet.

**Comment by lanesket** [comment, marked as answer]
```python
predictor = detection_predictor("db_resnet50", pretrained=False)
weights = torch.load('../weights/db_resnet50-ac60cadc.pt', map_location=device)
predictor.model.load_state_dict(weights)
```

**Reply by Rilay9** [reply]
I'm trying to use the ocr predictor and it seems to try to download even if pretrained is set to false. Any thoughts?

**Reply by Rilay9** [reply]
Detection predictor also throws an error if offline.
